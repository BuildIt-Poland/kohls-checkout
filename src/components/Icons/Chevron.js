import React from 'react';

function Chevron({ className }) {
  return (
    <svg className={className} xmlns="http://www.w3.org/2000/svg" width="19" height="12">
      <path fill="#666666" d="M19 2.6L16.5.1l-7 7-7-7L.1 2.6l9.4 9.5z" />
    </svg>
  );
}

export default Chevron;
