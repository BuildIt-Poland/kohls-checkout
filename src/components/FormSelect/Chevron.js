import React from 'react';
import styled from 'styled-components';

import { zIndexDefault } from '../../styles/designTokens';
import { Chevron as ChevronIcon } from '../Icons';

const Wrapper = styled.span`
  position: absolute;
  top: 3.6rem;
  right: 1rem;
  z-index: ${zIndexDefault};
  pointer-events: none;
`;

function Chevron() {
  return (
    <Wrapper>
      <ChevronIcon />
    </Wrapper>
  );
}

export default Chevron;
