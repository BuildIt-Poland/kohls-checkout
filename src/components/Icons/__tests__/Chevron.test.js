import React from 'react';
import { create } from 'react-test-renderer';

import Chevron from '../Chevron';

describe('COMPONENT - Icons Chevron', () => {
  it('renders correctly', () => {
    const component = create(<Chevron fill={'#222222'} />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
