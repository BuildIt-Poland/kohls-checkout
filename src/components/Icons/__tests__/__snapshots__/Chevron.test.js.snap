// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Icons Chevron renders correctly 1`] = `
<svg
  height="12"
  width="19"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M19 2.6L16.5.1l-7 7-7-7L.1 2.6l9.4 9.5z"
    fill="#666666"
  />
</svg>
`;
